lib/index.js

```diff
@@ -107,6 +107,18 @@ class PDFExporter extends EventEmitter {
       throw msg
     }
 
+    // https://stackoverflow.com/a/46789486/30665
+    if (args.ignoreCertificateErrors) {
+      // SSL/TSL: this is the self signed certificate support
+      electronApp.on('certificate-error', (event, webContents, url, error, certificate, callback) => {
```

Comment:
I'm sorry, but I just realized that this listener is in the wrong spot.  `createJob` can be called repeatedly if electron-pdf is being used as server endpoint.  That would register multiple listeners which would be bad.  It must be moved to the `start` function.

Reply:
And you should be able to pass the args to `start` from inside of `charge.js`

Reply:
please make the args optional though, e.g. `start(args={})` so that it remains backwards compatible.

Reply:
Yep, it seems to fail on my end when accessing a page with a self-signed certificate and the `--ignoreCertificateErrors` option is not present. The page is loaded correctly when that option is present.